lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -901,7 +902,7 @@ def commit(order, message_type, retry_logic = nil, trace_number = nil)
             request.call(remote_url(:secondary))
           end
 
-        authorization = order.include?('<TokenTxnType>GT</TokenTxnType>') ? response[:safetech_token] : authorization_string(response[:tx_ref_num], response[:order_id])
+        authorization = authorization_string(response[:tx_ref_num], response[:order_id], response[:safetech_token], response[:card_brand])
```

Comment:
One final question regarding this line. Should we more safely fetch the values from the hash? My concern is if the gateway does not return one of these keys in the response that we'll fail on an index error.